packages/cli/src/api/formats/po.js

```diff
@@ -1,104 +1,113 @@
 // @flow
 import fs from "fs"
 import * as R from "ramda"
+import { format as formatDate } from "date-fns"
```

Comment:
Thank you for this 🙏

Reply:
And I was worried you would object against including whole new package because of a single format call :) I am not sure if it will find some other uses, there isn't probably much of date time operations needed.

Reply:
It's a backend package, I wouldn't be worried about extra dependency. It's definitely better than format date manually...